packages/brain/src/modules/apiClients/dappnodeSignerProover.ts

```diff
@@ -0,0 +1,32 @@
+import { StandardApi } from "./index.js";
+import path from "path";
+import {
+  Network,
+  DappnodeSigningProoverPostRequest,
+} from "@stakingbrain/common";
+
+export class DappnodeSigningProover extends StandardApi {
+  private dappnodeSignEndpoint = "/newSignature";
+
+  constructor(network: Network, validatorsMonitorUrl?: string) {
+    super(
+      {
+        baseUrl: validatorsMonitorUrl || "https://dappnode-signatures.io",
+      },
+      network
+    );
+  }
+
+  public async sendProofOfAttestation(
```

Comment:
I would rename to `sendProofsOfAttestationBatch` (or something similar) since we're sending `proofOfAttestations: DappnodeSigningProoverPostRequest[]` and not a single attestation